test/run_pass/test_process_send.js

```diff
@@ -1,21 +1,39 @@
 'use strict';
 var assert = require('assert');
+// json length is close to 19KB
 var data = require('./test_process_send.json');
+var dataStr = JSON.stringify(data)
+var name = process.send ? 'child' : 'parent'
+
+var equalTimes = 0;
+function equalData(msg) {
+  if (typeof msg === 'object') {
+    console.log('stringify msg')
+    msg = JSON.stringify(msg);
+    // assert.equal(msg.length, JSON.stringify(data).length);
+  } else {
+    console.log('string')
+  }
+  ++equalTimes;
+  assert.equal(msg.length, dataStr.length)
+}
+
+var obj = null;
 if (process.send) {
-  // send with callback
-  process.send(data, err => {
-    assert.equal(err, undefined);
-  });
-  // send without callback
-  process.send(data);
+  obj = process
 } else {
   var fork = require('child_process').fork;
-  var sendTimes = 0;
-  var child = fork(module.filename, [], {});
-  child.on('message', msg => {
-    var msgLength = JSON.stringify(msg).length;
-    var dataLength = JSON.stringify(data).length;
-    console.log(`received ${msgLength}bytes data ${++sendTimes} times`);
-    assert.equal(dataLength, msgLength);
-  });
+  obj = fork(module.filename, [], {});
 }
+obj.on('message', equalData);
+
+var times = 10
+var timer = setInterval(() => {
+  console.log(name, 'sent', times)
+  --times;
+  obj.send(data);
+  obj.send(dataStr);
+  if (times <= 0) {
+    clearInterval(timer);
+  }
+}, 500)
```

Comment:
This test is more like as a benchmark script, we actually need the unit tests, could you please add an case that just acquire the API works and move this to benchmark?